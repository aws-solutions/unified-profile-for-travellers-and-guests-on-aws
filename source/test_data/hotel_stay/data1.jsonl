{"objectVersion":16,"modelVersion":"1.2.0","id":"93PK2EGAAV","bookingId":"UWOWIRV4QI","guestId":"7564990300","lastUpdatedBy":"Leonard Stanton","createdBy":"River Hodkiewicz","currency":{"code":"LRD","name":"Liberia Dollar"},"firstName":"Trace","lastName":"Turcotte","email":"","phone":"","startDate":"2024-12-01","hotelCode":"LON04","revenue":[{"type":"Room Charge 2024-12-01","description":"King Room with Mini bar ( Super Saver rate)","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":163.21436,"date":"2024-12-01T00:00:00Z"},{"type":"Room Charge 2024-12-02","description":"King Room with Mini bar ( Super Saver rate)","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":353.43756,"date":"2024-12-02T00:00:00Z"},{"type":"restaurant","description":"","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":63.80513,"date":"2024-12-02T17:15:10.787790778Z"},{"type":"just-walk-out-purchase","description":"","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":98.32317,"date":"2024-12-01T12:32:24.617091104Z"},{"type":"restaurant","description":"","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":75.72443,"date":"2024-12-02T15:33:58.470850515Z"},{"type":"restaurant","description":"","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":66.69219,"date":"2024-12-02T17:17:32.806281921Z"}],"alternateProfileIds":[{"name":"trade","value":"877d3270-52b2-4037-9c86-b9a2973de0e5","description":"Consequatur et esse quod voluptatem.","lastUpdatedOn":"2005-02-10T12:02:58.218046Z"}],"lastUpdatedOn":"2024-12-02T02:08:56.554011Z","createdOn":"2024-12-02T02:03:34.981340Z"}
{"objectVersion":2,"modelVersion":"1.2.0","id":"CNM20QVCFL","bookingId":"UWOWIRV4QI","guestId":"7564990300","lastUpdatedBy":"Rhett Kreiger","createdBy":"Leilani Mayert","currency":{"code":"LRD","name":"Liberia Dollar"},"firstName":"Trace","lastName":"Turcotte","email":"","phone":"","startDate":"2024-12-01","hotelCode":"LON04","revenue":[{"type":"Room Charge 2024-12-01","description":"King Room with Mini bar ( Super Saver rate)","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":163.21436,"date":"2024-12-01T00:00:00Z"},{"type":"Room Charge 2024-12-02","description":"King Room with Mini bar ( Super Saver rate)","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":353.43756,"date":"2024-12-02T00:00:00Z"},{"type":"just-walk-out-purchase","description":"","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":76.40594,"date":"2024-12-02T23:55:02.901950026Z"},{"type":"restaurant","description":"","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":26.13541,"date":"2024-12-02T01:13:01.262882406Z"},{"type":"sodas","description":"","currency":{"code":"LRD","name":"Liberia Dollar"},"amount":88.22542,"date":"2024-12-02T16:43:05.189763776Z"}],"alternateProfileIds":[{"name":"trade","value":"877d3270-52b2-4037-9c86-b9a2973de0e5","description":"Consequatur et esse quod voluptatem.","lastUpdatedOn":"2005-02-10T12:02:58.218046Z"}],"lastUpdatedOn":"2024-12-01T23:16:36.795661Z","createdOn":"2024-12-01T10:31:29.387399Z"}
