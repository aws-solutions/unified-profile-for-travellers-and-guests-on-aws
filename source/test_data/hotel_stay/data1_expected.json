{"hotel_stay_revenue_items": [{"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-03-31", "description": "Doubel room with  ( Xmas special rate)", "amount": 1021.1031, "date": "2022-03-31T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-01", "description": "Doubel room with  ( Xmas special rate)", "amount": 1629.0879, "date": "2022-04-01T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-02", "description": "Doubel room with  ( Xmas special rate)", "amount": 1613.5128, "date": "2022-04-02T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-03", "description": "Doubel room with  ( Xmas special rate)", "amount": 1120.8956, "date": "2022-04-03T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-04", "description": "Doubel room with  ( Xmas special rate)", "amount": 678.1976, "date": "2022-04-04T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-05", "description": "Doubel room with  ( Xmas special rate)", "amount": 1428.8134, "date": "2022-04-05T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-06", "description": "Doubel room with  ( Xmas special rate)", "amount": 1065.3573, "date": "2022-04-06T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "sodas", "description": "", "amount": 36.357292, "date": "2022-04-09T07:12:14.155849356Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "just-walk-out-purchase", "description": "", "amount": 97.72647, "date": "2022-03-31T15:14:32.091311354Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "snack bar", "description": "", "amount": 16.057598, "date": "2022-04-08T11:16:28.459838636Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-03-31", "description": "Doubel room with  ( Xmas special rate)", "amount": 461.84955, "date": "2022-03-31T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-03-31", "description": "King Room with Balcony ( Xmas special rate)", "amount": 770.44086, "date": "2022-03-31T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "just-walk-out-purchase", "description": "", "amount": 20.227053, "date": "2022-04-08T18:02:01.66194897Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-03-31", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 3859.7646, "date": "2022-03-31T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-03-31", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 1622.797, "date": "2022-03-31T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-01", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 1674.2655, "date": "2022-04-01T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-01", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 1032.4093, "date": "2022-04-01T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-02", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 1996.3243, "date": "2022-04-02T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-02", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 1562.7429, "date": "2022-04-02T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-03", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 1549.0326, "date": "2022-04-03T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-03", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 448.4849, "date": "2022-04-03T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-04", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 4264.713, "date": "2022-04-04T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-04", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 1094.6526, "date": "2022-04-04T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-05", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 2401.1038, "date": "2022-04-05T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-05", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 1492.164, "date": "2022-04-05T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-06", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 4186.0205, "date": "2022-04-06T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-06", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 2106.1396, "date": "2022-04-06T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-07", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 4093.6833, "date": "2022-04-07T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-07", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 1262.2366, "date": "2022-04-07T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-08", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 4147.9863, "date": "2022-04-08T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-08", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 1887.2651, "date": "2022-04-08T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-09", "description": "Suite with Balcony,Mini bar ( Xmas special rate)", "amount": 2089.9456, "date": "2022-04-09T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "Room Charge 2022-04-09", "description": "Suite with Mini bar ( Best avaialble rate)", "amount": 990.86035, "date": "2022-04-09T00:00:00Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-04-05T18:10:52.962905435Z", "created_on": "2022-04-02T22:41:26.449933196Z", "last_updated_by": "Lila Kiehn", "created_by": "Heidi Lesch", "id": "V4L5QXVUPO", "booking_id": "9J0XMHX69E", "traveller_id": "8981850427", "currency_code": "HKD", "currency_name": "Hong Kong Dollar", "currency_symbol": "", "first_name": "Abe", "last_name": "Robel", "email": "elvamueller@adams.info", "phone": "751-184-106-9277", "start_date": "2022-03-31", "hotel_code": "ROM14", "type": "sodas", "description": "", "amount": 99.62977, "date": "2022-04-07T15:36:52.452031135Z", "last_updated_partition": "2022/04/05/18/10/52/962905435Z"}]}